Answers.md

```diff
@@ -1,9 +1,19 @@
 1. If you had to describe semantic HTML to the next cohort of students, what would you say?
 
+Semantic HTML gives meaning to the browser about what is contained within your website. This means that it helps Search Engine Optimization which is very valuable to any website. 
+
 2. Describe some differences between ```display: block;``` and ```display: inline;```.
+display: block gives you the ability to give height to your element. It also allows for application of the box model properties. 
+diplay: inline enables you to alter certain properties of an element while keeping that element in the same line that it was in before you altered it. 
```

Comment:
per our discussion, display: block is about taking up the whole width of the page, whereas display-inline allows boxes to be set next to each other in the flow